crates/test-utils/src/minicore.rs

```diff
@@ -1825,7 +1825,7 @@ macro_rules! impl_int {
     ($($t:ty)*) => {
         $(
             impl $t {
-                pub const fn from_ne_bytes(bytes: [u8; mem::size_of::<Self>()]) -> Self {
+                pub const fn from_ne_bytes(bytes: [u8; size_of::<Self>()]) -> Self {
```

Comment:
This needs to be undone, this is effectively a core library subset so it doesnt have its own prelude

Reply:
With `size_of` added to the prelude, this is fine.